docs/notes/migration_guide.rst

```diff
@@ -396,6 +399,9 @@ your code to reflect these changes:
 Step 9: Examples refactor
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
+.. TODO:: What to do here? Write which examples in 2 turned into which examples in 3?
+   A lot of the info seems too much.
+
```

Comment:
I have doubts the section itself is necessary. In any case, what would a user look for here? They would be trying to update the previous version I imagine, so the changes we made are not as important, right? I would imagine something like what Juan suggested, mention which examples in v2 became what examples in v3, to maintain their function. Note that it doesn't seem straightforward to me at all, given their naming changes.